File: SDKMobile/SDK - iOS/Source/Payment.m
//
//  Payment.m
//  PostWebView
//
//  Created on 9/9/11.
//  Copyright 2011 DineroMail. All rights reserved.
//

#import "Payment.h"


@implementation Payment
@synthesize paymentMethod, installments;

-(id) initWithPaymentMethod:(NSString*) pm withInstallment:(NSMutableArray*) insts
{
	self = [super init];
	if (self) {
		self.paymentMethod = pm;
		self.installments = insts;
	}
	return self;
}
@end

File: SDKMobile/SDK - Blackberry/Source/com/dineromail/blackberry/model/Payment.java
package com.dineromail.blackberry.model;

 
public class Payment {

	private PaymentMethod paymentMethod;

	private int[] installments;

	public Payment(PaymentMethod paymentMethod, int[] installments) {
		super();
		this.paymentMethod = paymentMethod;
		this.installments = installments;
	}

	public PaymentMethod getPaymentMethod() {
		return paymentMethod;
	}

	public void setPaymentMethod(PaymentMethod paymentMethod) {
		this.paymentMethod = paymentMethod;
	}

	public int[] getInstallments() {
		return installments;
	}

	public void setInstallments(int[] installments) {
		this.installments = installments;
	}

}
